### PR TITLE
v0.6 docs minor updates

### PR DESCRIPTION
**Pull request by harper-u3d**
This PR fixes a few minor issues with documentation from the `release-v0.6` branch:

- In `Migrating.md`, the steps to migrate and change log were combined.  I separated them and added a step on migrating the Academy.
- In `Background-Tensorflow.md` we referred to an external tutorial on Tensorflow but didn't mention our own guide, which I've linked.
- Minor capitalization fix in `Background-Machine-Learning.md`

### CODE REVIEW

File: docs/Migrating.md [hunk 1]
```diff
@@ -42,6 +27,25 @@ using the search bar of the `Hierarchy` window with the word `Agent`.
   [Inference Engine](Inference-Engine.md).
 * To use a `.tf` model, drag it inside the `Model` property of the `Learning Brain`
 
+#### Steps to Migrate
+
+* To update a scene from v0.5 to v0.6, you must:
```
Comment (eshvk) [inline docs/Migrating.md]: @vincentpierre Please second pair of eyes on this.

File: docs/Migrating.md [hunk 2]
```diff
@@ -42,6 +27,25 @@ using the search bar of the `Hierarchy` window with the word `Agent`.
...
+    ML-Agents` for each type of the Brain you plan to use, and put
+    the created files under a folder called Brains within your project.
+  * Edit their `Brain Parameters` to be the same as the parameters used
+    in the `Brain` GameObjects.
+  * Agents have a `Brain` field in the Inspector, you need to drag the
+    appropriate Brain ScriptableObject in it.
+  * Academies have a `Broadcast Hub` field in the inspector, which is
+    list of brains used in the scene.  You need to drag all `Brain`
+    ScriptableObjects used in your scene into entries in this list.
+* You will need to delete the previous TensorFlowSharp package
```
[the remark this answers is not in the document]
Comment (vincentpierre) [reply docs/Migrating.md]: No, the previous bullets are about the scene. This one is about the project.

File: docs/Migrating.md [hunk 3]
```diff
@@ -42,6 +27,25 @@ using the search bar of the `Hierarchy` window with the word `Agent`.
...
+  * Remove the `Brain` GameObjects in the scene. (Delete all of the
+    Brain GameObjects under Academy in the scene.)
+  * Create new `Brain` Scriptable Objects using `Assets -> Create ->
+    ML-Agents` for each type of the Brain you plan to use, and put
+    the created files under a folder called Brains within your project.
+  * Edit their `Brain Parameters` to be the same as the parameters used
+    in the `Brain` GameObjects.
+  * Agents have a `Brain` field in the Inspector, you need to drag the
+    appropriate Brain ScriptableObject in it.
+  * Academies have a `Broadcast Hub` field in the inspector, which is
```
Comment (vincentpierre) [inline docs/Migrating.md]: Replace `Academies have` with `Academy has`. The broadcast hub does not need to have all of the brains present in the scene. Only the ones that need to be broadcasted or controlled by the Python process.

File: docs/Migrating.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (harperj) [reply docs/Migrating.md]: 👍

File: docs/Migrating.md [hunk 4]
```diff
@@ -42,6 +27,25 @@ using the search bar of the `Hierarchy` window with the word `Agent`.
...
+    Brain GameObjects under Academy in the scene.)
+  * Create new `Brain` Scriptable Objects using `Assets -> Create ->
+    ML-Agents` for each type of the Brain you plan to use, and put
+    the created files under a folder called Brains within your project.
+  * Edit their `Brain Parameters` to be the same as the parameters used
+    in the `Brain` GameObjects.
+  * Agents have a `Brain` field in the Inspector, you need to drag the
+    appropriate Brain ScriptableObject in it.
+  * The Academy has a `Broadcast Hub` field in the inspector, which is
+    list of brains used in the scene.  You need to drag all `Brain`
```
[the remark this answers is not in the document]
Comment (harperj) [reply docs/Migrating.md]: Ah right, forgot that you gave me this feedback in person.  Reworded this.

**Comment by vincentpierre** [comment]
🚢 🇮🇹

**Pull request merged**